runtime/vm/JFRChunkWriter.cpp

```diff
@@ -107,7 +107,7 @@ VM_JFRChunkWriter::writeJFRMetadata()
 	}
 
 	/* add metadata size */
-	_bufferWriter->writeLEB128PaddedU32(dataStart, _bufferWriter->getCursor() - dataStart);
+	_bufferWriter->writeLEB128PaddedU32(dataStart, U_32(_bufferWriter->getCursor() - dataStart));
```

Comment:
please use (U_32)(_bufferWriter->getCursor() - dataStart), Im not sure all the compilers will support the notation you have

Reply:
Done, and since https://github.com/eclipse-openj9/openj9/pull/20040 is merged, also fixed conversion introduced in that PR.